standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/ObjectStore.java

```diff
@@ -2607,6 +2614,9 @@ public boolean addPartitions(String catName, String dbName, String tblName, List
         tabColumnGrants = this.listTableAllColumnGrants(catName, dbName, tblName);
       }
       List<Object> toPersist = new ArrayList<>();
```

Comment:
can `toPersist` be removed? when need to persist the objects via JDO, we can do like:
```java
 pm.makePersistentAll(mParts);
 if (!mPartPrivilegesList.isEmpty()) {
   pm.makePersistentAll(mPartPrivilegesList);
  }
 //...
 pm.flush();
```

Reply:
ok